source/extensions/filters/http/jwt_authn/jwt_cache.cc

```diff
@@ -0,0 +1,32 @@
+#include "extensions/filters/http/jwt_authn/jwt_cache.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace JwtAuthn {
+
+JwtCache::JwtCache(int cache_size) { jwt_cache_.setMaxSize(cache_size); }
```

Comment:
in the constructor, only create jwt_cache_ if cache is enabled.

if pass in jwt_cache_size is 0,  then use predefined 
```
constexpr int kJwtCacheDefaultSize = 100;
```

Reply:
Done